### ISSUE TITLE
Table crafting bot limitations

### ISSUE THREAD

**Issue by WJohn** [tgstation/tgstation#12267]
When table crafting a bot (tested with a securitron) you _must_ use a right cyborg arm in order to be able to craft it, it will not accept a left one. This might be the case with the other bots, too. Might require cyborg arms to be put into their own path so it accepts either one.

**Comment by Rinoka** [comment]
Decided to peer over this -- this effects all bots in tablecrafting other than ED209, who is a snowflake with many legs and no arms.

The obvious fix is to put both arms under a singular craft, but that could lead to wonkiness with robot creation and might be outright unnecessary.

It might be easiest to make it so the craft product makes an incomplete assembly that you finish with either arm, even if that's a bit snowflakey for tablecrafting.

(Still, it'd be a simple fix for a consistency issue that's been open for years.)

**Comment by PKPenguin321** [comment]
I feel this is more of a problem with tablecrafting than it is with how arms are pathed
You should be able to have alternate recipes for the same tablecrafting entry

**Comment by Rinoka** [comment]
Yeah, I agree -- the problem is that I can't see any good way to write in an "or" alternative to the recipe, sans maybe attempting to compact another list into the list so I can list while I list.

Because the alternatives are doing a complete copy of the recipe but with robot/larm instead (ugly, extra data if not coded right) or the path fix.

I think the assembly might be nicer, though, if only because it's less effort and resembles things like foodcode right now, since food is very "put in what you want" focused. (It's also more within the bounds of something someone like me could code.)

It is an interesting conundrum, at least.

**Comment by ExcessiveUseOfCobblestone** [comment]
The solution is to wrap the required items in a proc a la `PopulateContents()` and not a var.

**Comment by ExcessiveUseOfCobblestone** [comment]
or just scrap tablecrafting anything with any requirement for dynamic implementation :^)

**Comment by PKPenguin321** [comment]
or just allowing items to have alternate recipes

**Comment by dragomagol** [comment]
No longer seems to be an issue (all bots can be made with either arm)

**Comment by dragomagol** [comment]
Turns out it wasn't actually fixed, but the step-by-step method can take either.

**Comment by Razharas** [comment]
The relatively easy fix is to make is to wrap all the recipe req and part entries into lists and all the direct "==" type checking into "in" checking instead
No additional logic needed, just replacement

**Comment by Sealed101** [comment]
did #71779 affect this in any way?

**Comment by SyncIt21** [comment]
@Fikou i'm pretty sure you fixed this with #74971?

**Comment by SyncIt21** [comment]
Ok so i tried crafting a clown bot but the above linked PR doesn't exactly help with this problem because the crafting recipe is just bad and leaves no space for subtypes

![Screenshot (201)](https://user-images.githubusercontent.com/110812394/236639909-57ec284d-a900-4caa-b1fb-edfc9f9ade58.png)

Like why is it `arm/right/robot`? it should be` arm/robot/right` which would allow for subtypes but that's a whole PR in itself so this bug lives another day

**Comment by Fikou** [comment]
@deigodoraflies456 bro whats with the reacts
